README.md

```diff
@@ -1,5 +1,8 @@
-# Trampss TodoMVC Example
-[![CircleCI](https://circleci.com/gh/Trampss/todomvc.svg?style=svg)](https://circleci.com/gh/Trampss/todomvc) [![Coverage Status](https://coveralls.io/repos/github/Trampss/todomvc/badge.svg?branch=master)](https://coveralls.io/github/Trampss/todomvc?branch=master)
+# Our best way to make TodoMVC Example
```

Comment:
Nope

Reply:
"Todo MVC with ReactJS + redux-saga + Redux" ?

Reply:
If you are proud and want to show it, maybe something like `with Alakarte stack`.
And we could make a link to our talk on the title later ?